src/Chapter2.hs

```diff
@@ -336,7 +336,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b xs
+  | a < 0 || b < 0 || a > b = []
+  | otherwise = drop a (take (b+1) xs)
```

Comment:
Great!
Usualy in Haskell all the operators are separated by spaces from its arguments:

```suggestion
  | otherwise = drop a (take (b + 1) xs)
```